tests/manual-test-cases/Group14-Longevity/14-1-Longevity.robot

```diff
@@ -29,11 +29,11 @@ Longevity
     :FOR  ${idx}  IN RANGE  0  48
     \   ${rand}=  Evaluate  random.randint(10, 50)  modules=random
     \   Log To Console  \nLoop: ${idx}
-    \   Install VIC Appliance To Test Server  debug=%{DEBUG_VCH_LEVEL}  additional-args=%{STATIC_VCH_OPTIONS} %{SYSLOG_VCH_OPTION}
+    \   Install VIC Appliance To Test Server  debug=%{DEBUG_VCH_LEVEL}  additional-args=%{STATIC_VCH_OPTIONS} %{SYSLOG_VCH_OPTION} --container-name-convention longevity-{id}
```

Comment:
Are there any tests in the longevity set that use govc to inspect/interact with cVMs? If so then this should perhaps have `longevity_date_{name}-{id}` so its easy to update the tests.

Can we have a date or run identifier in the template so we can tie them back to a specific longevity run? One of the issues with the leaking cVMs atm is that we cannot tell when they were created as they're re-indexed from the host

Reply:
done, there are not any tests in longevity that use govc to interact with cVMs.